src/studio/nuxeo-studio/nuxeo-studio-designer-git-access.md

```diff
@@ -0,0 +1,74 @@
+To activate it, simply request it through a support ticket by indicating:
+- Who should be given access
+- Which access these people should obtain (read or write)
+- On which project(s)
+
+## Feature Scope and Restrictions
+
+ Nuxeo Studio Designer Git access has been designed to allow for productivity, while keeping sane restrictions for security and ease of use reasons.
+
+### What's possible
+- Access to Nuxeo Studio Git repositories through HTTPS protocol
+- Launch git commands on your repositories, with some restrictions
+- Automatic refresh of the Studio project cache when git commands have been launched
+
+### Restrictions
+- Only **fast-forward changes** can be pushed.
+- Only **Studio Designer files** can be managed. Changes to the Modeler side of your project will automatically be rejected as we currently have no way to ensure the validity of your configuration made through this mean.
+- **We don't and will never allow rewriting history** to make sure you can always revert your project to a working state. Push force for instance will be rejected.
+- **Creating / Deleting Studio branches from Git is not allowed**. You need to use Studio for this. Reason is Nuxeo Studio manages business logic on top of them.
+- **Creating / Updating / Deleting tags from Git is not allowed**. Releases can be created through Studio’s interface or using a REST API. Again, this is tied to Studio managing business logic on top of this.
+
+### Known Limitations
+- No SSH protocol access.
+
+## How to Use Git Access
+### Cloning Your Project
+Cloning your repositories can be done using your Nuxeo Online Services credentials. As a prerequisite, a Git client should be installed on your computer.
+
+Once logged in to Nuxeo Online Services:
+1. Visit the `My Applications` tab
+2. Copy the Git url to clone your project
+3. Clone your project using your Git client (typically `git clone [project URL]`)
+
+{{#> callout type='info' heading='Password Storage'}}
+If you want to avoid typing your password repeatedly, we recommend generating a [token]({{page page='token-management'}}) in your Nuxeo Online Services portal and storing it using Git's standard `.netrc` file mechanism.
+{{/callout}}
+
+### Making Changes
+The recommended flow is the following:
+
+1. In Nuxeo Studio, create a branch for your changes. It will be named `feature/your-branch-name`.
+1. Pull your changes
+1. Checkout the `feature/your-branch-name` branch
+1. Make some changes
+1. Commit and push your changes
+
+{{#> callout type='info' heading='Branch Usage'}}
+It is also possible to use any existing feature branch (feature branches are all named `feature/branch-name`) or master to push your changes. Nevertheless, we recommend using a dedicated branch to have more control.
+{{/callout}}
+
+### Checking and Integrating the Result
+Have a look at your changes in Studio:
+
+1. Checkout the feature/your-branch-name branch if needed
+1. Pull changes. Studio checks every 10 seconds for new commits to pull.
```

Comment:
I wouldn't tell 10 seconds, rather `Studio checks regularly for new commits to pull`. This would prevent any doc inconsistency if we happen to change the value :)

Reply:
Makes sense, changing this.